db/db_impl/db_impl.cc

```diff
@@ -4391,10 +4391,14 @@ Status DBImpl::CheckConsistency() {
       }
       files_by_directory[md.db_path].push_back(fname);
     }
+
+    IOOptions io_opts;
+    io_opts.SetProperty("list_files_only", "true");
```

Comment:
Why do we need to define it as a property? As the comments indicate, the property_bag is opaque to RocksDB and is intended for passing options directly from a 3rd party fFileSystem user to the FileSystem. For options that RocksDB is going to use, why can't we just define a bool in IOOptions?

Reply:
Since it's been used by `GetChildren` API only that's why I added it in `property_bag`. Since it violates the agreement, it can add it to `IOOptions` as `bool`.